test/cppsuite/src/main/configuration.cpp

```diff
@@ -82,10 +82,7 @@ configuration::configuration(const std::string &test_config_name, const std::str
     logger::log_msg(LOG_INFO, "Full config: " + _config);
 
     int ret =
-      wiredtiger_test_config_validate(nullptr, nullptr, test_config_name.c_str(), _config.c_str());
```

Comment:
Why are we removing this? The cppsuite uses it to validate the config that it generates...

Reply:
@korteland

Reply:
So your plan is that instead of erroring while validation checking it we'll error when trying to fetch items from it?

Reply:
Yes, that's right. It's less convenient - the original code wasn't pointless - but it takes an entire function out of the public API.

Reply:
Yeah okay I'll let this one go. Btw I still think one PR is cleaner for such a small set of changes instead of two #10261